src/promptflow-evals/promptflow/evals/_common/utils.py

```diff
@@ -28,3 +40,19 @@ def get_harm_severity_level(harm_score: int) -> str:
         if harm_score_range[0] <= harm_score <= harm_score_range[1]:
             return harm_level.value
     return np.nan
+
+
+def nltk_tokenize(text: str) -> List[str]:
```

Comment:
Just curious what is this for ?

Reply:
It's for splitting the sentense into tokens. I have a unit-test which can demosntrate what it does.